.travis.yml

```diff
@@ -25,12 +35,15 @@ matrix:
           env: BEFORE="./bin/travis/setupphpunit.sh" SYMFONY_VERSION="^3.2" TEST_CMD="./vendor/bin/phpunit -c phpunit.xml" AFTER_SUCCESS='echo "After success"'
         - php: 5.6
           env: BEFORE='./bin/travis/setupphpunit.sh' TEST_CMD="./vendor/bin/phpunit -c phpunit.xml" AFTER_SUCCESS='echo "After success"'
+    allow_failures:
```

Comment:
actually, I realized on Friday that https://github.com/ezsystems/PlatformUIBundle/pull/854 totally breaks Behat Tests as with this patch there's no more app to be tested :)